source/includes/_applications.md

```diff
@@ -369,3 +369,33 @@ In order to delete a closed or hired Job Application you must use the following
 `DELETE /v1/companies/{companyId}/applications/{jobApplicationId}`
 
 If the request succeeds, the api will return a 200 response with no body.
+
+## Edit a JobApplication
+
+This endpoint is used to edit a job application's information. It can also be used to edit and delete associated files and profile pictures from storage.
+To do this provide an empty array in the `files` field. When updating the job application any existing file that is left out in the update request, will be deleted.
```

Comment:
To do this provide an empty array in the `files` field. When updating the job application any previously existing file that is left out in the update request, will be deleted.